modules/aggregation-layers/src/hexagon-layer/hexagon-aggregator.js

```diff
@@ -36,17 +36,26 @@ export function pointToHexbin({data, radius, getPosition}, viewport) {
 
   // add world space coordinates to points
   const screenPoints = [];
+  let position;
```

Comment:
Declare the variable inside the for loop if it's not used across iterations.

Reply:
Addressed in latest change